pkg/filter/http/dubboproxy/dubbo.go

```diff
@@ -125,8 +124,8 @@ func (f *Filter) Decode(hc *pixiuHttp.HttpContext) filter.FilterStatus {
 
 	if len(splits) != 3 {
 		logger.Info("[dubbo-go-pixiu] http path pattern error. path pattern should be http://127.0.0.1/{application}/{service}/{method}")
-		bt, _ := json.Marshal(pixiuHttp.ErrResponse{Message: "http path pattern error"})
-		hc.SendLocalReply(http.StatusBadRequest, bt)
+		errResp := pixiuHttp.BadRequest.WithError(fmt.Errorf("http path pattern error"))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format

```suggestion
		errResp := pixiuHttp.BadRequest.WithError(errors.New("http path pattern error"))
```